Source/Flow/Public/Nodes/FlowNode.h

```diff
@@ -50,30 +45,12 @@ class FLOW_API UFlowNode : public UObject, public IVisualLoggerDebugSnapshotInte
 
 	UPROPERTY()
 	TArray<TSubclassOf<UFlowAsset>> DeniedAssetClasses;
-
```

Comment:
Much of the FlowNode properties moved to FlowNodeBase, with the notable exceptions of those having to do with pins and focus.

Also, many of the functions moved to FlowNodeBase or into the Interfaces' implementations.